migrate-dao.js

```diff
@@ -199,7 +199,7 @@ async function migrateDAO ({ arcVersion, web3, spinner, confirm, opts, migration
     founders.map(({ reputation }) => web3.utils.toWei(reputation !== undefined ? reputation.toString() : '0'))
   ]
 
-  const initFoundersBatchSize = 20
+  const initFoundersBatchSize = 100
```

Comment:
does this change related to the transition script?

Reply:
Partly so, because the transition requires usually a lot of founders, and I had to modify this part anyway to allow that.